src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/PersonalisationProvider.java

```diff
@@ -221,4 +220,54 @@ private static String readStringCaseField(final AsylumCase asylumCase, final Asy
         final Optional<String> optionalFieldValue = asylumCase.read(caseField, String.class);
         return optionalFieldValue.isPresent() && !optionalFieldValue.get().isEmpty() ? optionalFieldValue.get() : defaultIfNotPresent;
     }
+
+    public String getFtpaDecisionTemplateId(AsylumCase asylumCase,
+                                            GovNotifyTemplateIdConfiguration govNotifyTemplateIdConfiguration, YesOrNo applicant) {
+        FtpaAppellantDecisionOutcomeType ftpaAppellantDecisionOutcomeType = asylumCase
+            .read(FTPA_APPELLANT_DECISION_OUTCOME_TYPE, FtpaAppellantDecisionOutcomeType.class)
+            .orElseThrow(() -> new IllegalStateException("ftpaApplicationDecision is not present"));
+
+        if (applicant.equals(YesOrNo.YES)) {
+            return getApplicantTemplateId(govNotifyTemplateIdConfiguration, ftpaAppellantDecisionOutcomeType);
+        } else {
+            return getOtherPartyTemplateId(govNotifyTemplateIdConfiguration, ftpaAppellantDecisionOutcomeType);
+        }
+    }
+
+    public String getOtherPartyTemplateId(GovNotifyTemplateIdConfiguration govNotifyTemplateIdConfiguration, FtpaAppellantDecisionOutcomeType ftpaAppellantDecisionOutcomeType) {
+        if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_GRANTED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationGrantedOtherParty();
+        } else if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_PARTIALLY_GRANTED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationPartiallyGrantedOtherParty();
+        } else if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_REFUSED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationRefusedOtherParty();
+        } else {
+            return govNotifyTemplateIdConfiguration.getApplicationNotAdmittedOtherParty();
+        }
+    }
+
+    public String getApplicantTemplateId(GovNotifyTemplateIdConfiguration govNotifyTemplateIdConfiguration, FtpaAppellantDecisionOutcomeType ftpaAppellantDecisionOutcomeType) {
+        if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_GRANTED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationGrantedApplicant();
+        } else if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_PARTIALLY_GRANTED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationPartiallyGrantedApplicant();
+        } else if (ftpaAppellantDecisionOutcomeType.toString().equals(FtpaAppellantDecisionOutcomeType.FTPA_REFUSED.toString())) {
+            return govNotifyTemplateIdConfiguration.getApplicationRefusedApplicant();
+        } else {
+            return govNotifyTemplateIdConfiguration.getApplicationNotAdmittedApplicant();
+        }
+    }
+
```

Comment:
Could possibly eliminate some code duplication between these two methods. May be worth looking to refactor here if possible.